mlflow/metrics/base.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, List
+
+from mlflow.models import EvaluationMetric as EvaluationMetric
```

Comment:
do we need to alias this import? otherwise can we just do this
```suggestion
from mlflow.models import EvaluationMetric
```

Reply:
without the `as EvaluationMetric` the formatter will remove these two lines since the imports are "not used"

Reply:
I think we can import `EvaluationMetric` and `make_metric` from `mlflow.models` in `mlflow/metrics/__init__.py` (unless there is a reason we want to expose them in `mlflow/metrics/base.py`).